apps/web/pages/v2/settings/my-account/general.tsx

```diff
@@ -34,25 +34,28 @@ const SkeletonLoader = () => {
 
 interface GeneralViewProps {
   localeProp: string;
+  user: inferQueryOutput<"viewer.me">;
 }
 
 const WithQuery = withQuery(["viewer.public.i18n"], { trpc: { context: { skipBatch: true } } });
 
 const GeneralQueryView = () => {
+  const { data: user, isLoading } = trpc.useQuery(["viewer.me"]);
```

Comment:
@zomars I think it would be a good idea to document this useForm `defaultValues` and tRPC scenario that ends up initializing form with undefined values.